public/app/features/alerting/unified/hooks/useCombinedRule.ts

```diff
@@ -141,15 +149,15 @@ export function useCombinedRule({ ruleIdentifier, limitAlerts }: Props): Request
 
     const rulerConfig = rulerRuleGroup ? { [ruleLocation.namespace]: [rulerRuleGroup] } : {};
 
-    const combinedNamespaces = combineRulesNamespaces(ruleSource, promRuleNs, rulerConfig);
+    const combinedNamespaces = combineRulesNamespace(ruleSource, promRuleNs, folderName ?? '', rulerConfig);
```

Comment:
If we apply the changes above then, the changes to this function should not be needed